doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/air-system-compound-component-groups.tex

```diff
@@ -2545,3 +2553,23 @@ \subsubsection{References}\label{references-1}
+
+This section describes the EIR formulated model for water-to-water heat pumps. (Object names: HeatPump:WaterToWater:EIR:Cooling \& HeatPump:WaterToWater:EIR:Heating). In general, these heat pump objects are treated the same as the other heat pump models by the plant loop.  Even though a heat pump is generally a single load coil and a single source coil with a reversing valve, in EnergyPlus, the paradigm is to split the operation into two separate coils, a heating and a cooling.  It is certainly possible to connect the load side of both of these to a single plant loop if the controls are established properly.  The source side of the coils are often placed on a single loop with some form of condensing supply, either a ground loop heat exchanger or a cooling tower (or a combination of the two).
+
+Note that this component is currently a constant flow device (it will always request its full design flow from the plant), with a future addition to make it respond to outlet setpoint in order to vary the flow request.
+
+The bookkeeping part of the heat pump code with regards to flow requests and nodal conditions is identical to other components, so this section will only briefly discuss the mathematical formulation of the model.
+
+Once load and source flow rates have been established for the component using the SetComponentFlowRate interface to the plant, and inlet temperatures for each side have been read, a function calculates the response of the unit to these conditions.  The process to evaluate the outlet state is as follows:
+
+\begin{enumerate}
+	\item Retrieve load side outlet temperature setpoint
+	\item Evaluate the capacity modifier function using the source inlet temperature and the load side setpoint temperature as the projected outlet temperature ($f_{cap} = f(T_{out,load,set}, T_{in,src}) $)
+	\item Calculate available load side capacity using the reference value and the modifier ($\dot{q}_{load,avail} = \dot{q}_{reference} \cdot f_{cap}$)
+	\item Calculate the part load ratio using the current plant-dispatched load (constrained between zero and one) ($PLR = \frac{\dot{q}_{plant}}{\dot{q}_{avail}}$)
+	\item Calculate the actual operating load side heat transfer and the load side outlet temperature ($\dot{q}_{load} = \dot{q}_{avail}PLR$, $T_{out,load} = T_{in,load} \pm \frac{\dot{q}_{load}}{\dot{m}_{load}C_{p,load}}) $) -- note the plus/minus refers to how the value will be added in heating mode and subtracted in cooling mode.  This is required because the sign convention is not consistent and a positive value for calculated heat rate will be obtained for both units under normal conditions.
+	\item Evaluate power usage by evaluating the EIR curves using the source inlet temperature, actual load side outlet temperature, and the current part load ratio ($f_{eir,t} = f(T_{out,load,set}, T_{in,src})$, $f_{eir,plr} = f(PLR)$, $P = P_{reference} f_{eir,t} f_{eir,plr}$)
```

Comment:
I don't see P_reference defined anywhere.

Reply:
Added a note that it is the reference power usage.